source/runbooks/disabling-network-connections-if-needed.html.md.erb

```diff
@@ -0,0 +1,99 @@
+---
+owner_slack: "#modernisation-platform"
+title: Disabling network access if required
+last_reviewed_on: 2024-02-14
+review_in: 6 months or when is resolved.
+---
+
+# <%= current_page.data.title %>
+
+## Overview
+
+There may be occasions where there is a network attack on Modernisation Platform environments that could require a way of stopping access. This document lists the possible solutions to the issue.
+
+[NOTE]
+
+> All the steps below assume you can access the system through the AWS console and do not require changes to the terraform code. This is based on initial speed of access. There may be times when amending the code is quicker but this is not included here.
+
+## Establish if there is an attack or unwanted access
+
+Establish if there is unwanted access or an attack taking place by checking the details on cloudwatch in both core-network-services and core-logging. It should be obvious from the contents of the logs.
+
+On core-network-services examine the contents, for production services, of
+
+- fw-live-data-inline-inspection-logs-ulpiqfaq
+- live\*data-vpc-flow-logs-qoa52v4z
+- external-inspection-flow-logs
+
+or the following for non-production services
+
+- non_live_data-vpc-flow-logs-fro8eqat
+- fw-non-live-data-inline-inspection-logs-ks3lnpsk
+
+On core-logging examine the contents, for production services, of
+
+- live_data-vpc-flow-logs
+  or the following for non-production services
+- non_live_data-vpc-flow-logs
+
+## If an issue has been established
+
+You need to then logon to the environment for which there is an issue. For example if it is on NOMIS Combined Reporting Preproduction (nomis-combined-reporting-production) then connect to the core-vpc-preproduction. In this case it would be the hmpps VPC.
+
+1. Connect to the service involved.
+2. INITIALLY detatch the environment from the VPC using Actions and Delete VPC. It might not be possible to delete this as there will be networked interfaces attached which need to be noted and deleted. In this case amending the code may be more appropriate. In the above example there are 166 preproduction network interfaces to be removed.
```

Comment:
Deleting any VPC is not desired.

Reply:
Maybe put that as the last - if everything else fails - option?